components/listbox/toolbar.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Toolbar
+page_title: ListBox - Toolbar
+description: Description of the Blazor ListBox toolbar and its built-in buttons. How to define custom toolbar buttons.
+slug: listbox-toolbar
+tags: telerik,blazor,listbox
+published: True
+position: 10
+---
+
+# ListBox Toolbar
+
+The ListBox toolbar can render built-in and custom tools. The built-in tools are buttons that reorder and remove items, or transfer items from and to another ListBox component. This article describes the built-in tools and shows how to add custom tools or customize the toolbar.
+
+
+## Built-in Tools
+
+By default, the ListBox displays all its built-in tools in the order below. Use the respective tool tag if you need to define a tool explicitly in a [custom toolbar configuration](#toolbar-configuration).
+
+Each button becomes enabled when it can be used, for example, when the user selects items. Each built-in button fires an event and the app must implement the required data source operation.
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Tool Name | Tool Tag | Event On Click | Description |
+| --- | --- | --- | --- |
+| Move Up | `<ListBoxToolBarMoveUpTool />` | [`OnReorder`]({%slug listbox-events%}#onreorder) | Moves the selected item(s) down by one position. |
+| Move Down | `<ListBoxToolBarMoveDownTool />` | [`OnReorder`]({%slug listbox-events%}#onreorder) | Moves the selected items up by one position. |
+| Transfer To | `<ListBoxToolBarTransferToTool />` | [`OnTransfer`]({%slug listbox-events%}#ontransfer) | Moves the selected items to a connected ListBox component. |
+| Transfer From | `<ListBoxToolBarTransferFromTool />` | [`OnTransfer`]({%slug listbox-events%}#ontransfer) | Moves the selected items from a connected ListBox component to the current one. |
+| Transfer All To | `<ListBoxToolBarTransferAllToTool />` | [`OnTransfer`]({%slug listbox-events%}#ontransfer) | Moves all items to a connected ListBox component. |
+| Transfer All From | `<ListBoxToolBarTransferAllFromTool />` | [`OnTransfer`]({%slug listbox-events%}#ontransfer) | Moves all items from the connected ListBox component to the current one. |
+| Remove | `<ListBoxToolBarRemoveTool />` | [`OnRemove`]({%slug listbox-events%}#onremove) | Removes the selected item(s). |
+
+>tip The descriptions above mention the typical user operation that each button implies. However, the exact operation depends entirely on the business requirements and event handler implementation. For example:
```

Comment:
I suggest removing the tip styling. Currently, you have half of the logically connected content in a tip and another half in a bullet list outside the tip. This creates a confusing incoherence.